<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuGUI</class>
 <widget class="QDialog" name="MenuGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="lay_superior">
     <item>
      <widget class="QGroupBox" name="lay_conexao">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox{
border-width: 1px;
margin-top: 10px;
padding: 1px;
border-style: solid;
border-color: #CCC;
}
 
QGroupBox::title{
subcontrol-origin: margin;
padding: 0 7px;
left: 5px;
}</string>
       </property>
       <property name="title">
        <string>Dados da Conexão</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="3">
         <widget class="QLineEdit" name="edt_senha">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="edt_ip"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="lbl_schema">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Schema:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="lbl_ip">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>IP:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="edt_schema">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="lbl_senha">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Senha:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cmb_db">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string>SQLITE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>POSTGRESQL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ORACLE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MYSQL</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbl_db">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Database:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="edt_usuario"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="lbl_usuario">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Usuário:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="lay_botoes">
       <item>
        <widget class="QPushButton" name="btn_conectar">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Conectar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_desconectar">
         <property name="text">
          <string>Desconectar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_executar">
         <property name="text">
          <string>Executar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QPlainTextEdit" name="txt_sql"/>
     <widget class="QTableView" name="tbl_sql"/>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edt_ip</tabstop>
  <tabstop>edt_schema</tabstop>
  <tabstop>edt_senha</tabstop>
  <tabstop>btn_conectar</tabstop>
  <tabstop>btn_desconectar</tabstop>
  <tabstop>btn_executar</tabstop>
  <tabstop>txt_sql</tabstop>
  <tabstop>tbl_sql</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
